src/main/java/ch/bfh/btx8081/w2019/green/alzman/view/DashboardView.java

```diff
@@ -15,7 +15,7 @@
 import com.vaadin.flow.server.PWA;
 
 
-@Route("")
```

Comment:
Route should point to the DashboardView, it is empty here

Reply:
gratuliere :)